packages/components/tree-v2/src/hooks/useCheck.ts

```diff
@@ -0,0 +1,220 @@
+import { nextTick, ref, watch } from 'vue'
+import { NODE_CHECK_CHANGE, NODE_CHECK } from '../emits'
+import type { Ref } from 'vue'
+import type {
+  ITreeProps,
+  TreeKey,
+  TreeNode,
+  Tree,
+  TreeNodeData,
+} from '../tree.type'
+
+const DELETE = 'delete'
+const ADD = 'add'
```

Comment:
Can we make this a enum named `operation`

```ts
const enum operation = {
  ADD = 'add',
  DELETE = 'delete'
}
```

Reply:
IMO `Operation` maybe better